plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -90,6 +84,11 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_TOPN_PUSHDOWN_WITH_VARCHAR:
                 return true;
 
+            case SUPPORTS_AGGREGATION_PUSHDOWN_STDDEV:
+            case SUPPORTS_AGGREGATION_PUSHDOWN_VARIANCE:
+            case SUPPORTS_AGGREGATION_PUSHDOWN_COVARIANCE:
+            case SUPPORTS_AGGREGATION_PUSHDOWN_CORRELATION:
+            case SUPPORTS_AGGREGATION_PUSHDOWN_REGRESSION:
```

Comment:
don't glue them to `case SUPPORTS_JOIN_PUSHDOWN`
```suggestion
            case SUPPORTS_AGGREGATION_PUSHDOWN_REGRESSION:
                return true;
                
```